### ISSUE TITLE
1.0.2 pip installation failed 

### ISSUE THREAD

**Issue by gaeldb** [Niklas9/django-unixdatetimefield#8]
Hi @Niklas9 

We have issue with version 1.0.2 when installing from pip.

> [root@centos7 ~]# pip3.6 install django-unixdatetimefield==1.0.2
Collecting django-unixdatetimefield==1.0.2
  Using cached https://files.pythonhosted.org/packages/ef/c1/ce364a9097c50d8073484ee13475eb6e191bc3206001f54e9b4d1e5ac9d2/django-unixdatetimefield-1.0.2.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-7j8t9mrv/django-unixdatetimefield/setup.py", line 4, in <module>
        with open('README.md', 'r') as f:
    FileNotFoundError: [Errno 2] No such file or directory: 'README.md'

We checked out the tar.gz provided by pythonhosted.org and the README.md file is missing.
Version 1.0.1 works well.

Thanks,

Gaël

**Comment by Niklas9** [comment]
Thanks for letting me know and sorry about this @gaeldb , working on a fix

**Comment by Niklas9** [comment]
Fixed in 1.0.3, commit https://github.com/Niklas9/django-unixdatetimefield/commit/562fb0a8a2dfbda854cc1d5916b4b2982b9fe2cc.
I switched from reStructuredText to Markdown for the readme, and forgot to update the manifest :( Thanks again for the ping.

```
$ pip3 install --no-cache-dir django-unixdatetimefield==1.0.3
Collecting django-unixdatetimefield==1.0.3
  Downloading https://files.pythonhosted.org/packages/e1/51/7fb314888c14256ce42f749afaa833b53355aab61d96ccdf50c934d87291/django-unixdatetimefield-1.0.3.tar.gz
Collecting django (from django-unixdatetimefield==1.0.3)
  Downloading https://files.pythonhosted.org/packages/c7/87/fbd666c4f87591ae25b7bb374298e8629816e87193c4099d3608ef11fab9/Django-2.1.7-py3-none-any.whl (7.3MB)
    100% |################################| 7.3MB 7.4MB/s
Collecting pytz (from django->django-unixdatetimefield==1.0.3)
  Downloading https://files.pythonhosted.org/packages/61/28/1d3920e4d1d50b19bc5d24398a7cd85cc7b9a75a490570d5a30c57622d34/pytz-2018.9-py2.py3-none-any.whl (510kB)
    100% |################################| 512kB 9.7MB/s
Installing collected packages: pytz, django, django-unixdatetimefield
  Running setup.py install for django-unixdatetimefield ... done
Successfully installed django-2.1.7 django-unixdatetimefield-1.0.3 pytz-2018.9
```
